openmls/src/test_utils/test_framework/mod.rs

```diff
@@ -108,7 +110,7 @@ pub struct MlsGroupTestSetup {
 // references, as we can't mutably reference the keystore.
 //
 //   * Note, that (to my knowledge) we can't have the keystore live in a
-//   refcell, because then the references are based on the local `borrow()`ed
+//   refcell, because then the references are based on the local `read().expect("An unexpected error occurred.")`ed
```

Comment:
accidental replace? 😄 I think this whole comment is pretty much outdated, so feel free to remove it all.